pkg/scheduler/service/cleaner.go

```diff
@@ -61,3 +125,99 @@ func (c *cleaner) purgeReconciliations(transition *ClusterStatusTransition, conf
 		}
 	}
 }
+
+func (c *cleaner) getLatestReconciliations(latestEntitiesCount int, transition *ClusterStatusTransition) ([]*model.ReconciliationEntity, error) {
+	return transition.ReconciliationRepository().GetReconciliations(&reconciliation.Limit{Count: latestEntitiesCount})
```

Comment:
Should we set order by? otherwise, the returned results might not be the latest.

Reply:
OK, resolved, the implementation of limit contains order by
```
func (l *Limit) FilterByQuery(q *db.Select) error {
	q.OrderBy(map[string]string{"Created": "DESC"}).Limit(l.Count)
	return nil
}